public/app/features/auth-config/ProviderConfigPage.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { connect, ConnectedProps } from 'react-redux';
+
+import { AppEvents, NavModelItem, SelectableValue } from '@grafana/data';
+import { getAppEvents, getBackendSrv, isFetchError } from '@grafana/runtime';
+import { Button, Field, Input, InputControl, LinkButton, Select, Stack, Switch } from '@grafana/ui';
+import { FormPrompt } from 'app/core/components/FormPrompt/FormPrompt';
+import { Page } from 'app/core/components/Page/Page';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+
+import { StoreState } from '../../types';
+
+import { FieldData, fieldMap, fields } from './fields';
+import { loadSettings } from './state/actions';
```

Comment:
This page should use the SSO settings api (`GET /api/v1/sso-settings/github`) for fetching the data, not the admin/api/settings.

Reply:
`loadSettings` loads both the SSO settings and admin settings. The admins settings are necessary to load SAML settings.

Reply:
Ohh, sorry, I didn't notice that! Thanks!